docs/core/docker/intro-net-docker.md

```diff
@@ -1,11 +1,12 @@
 ---
-title: Introduction to .NET and Docker
-description: Understanding Docker and .NET Core
+title: Introduction to Docker - .NET Core
+description: This article provides an introduction and overview to Docker in the context of a .NET Core application.
 author: jralexander
 ms.author: johalex
 ms.date: 11/06/2017
-ms.custom: mvc
+ms.custom: "mvc", "seodec18"
```

Comment:
This should be:

```yaml
ms.custom: "mvc, seodec18"
```

It's not really YAML. It's a string.

Reply:
Doug Erickson said in the chat on the SEO training that you need to use multiple strings separated with a , to prevent it from being seen as a single string (they want multiple string values)

Reply:
Yes, but then he and Carl Rabeler investigated. It needs to be a single YAML string with the values separated by commas.

Reply:
I ran it though an online yaml validator and it hated the `", "` syntax.. So you're probably right. I'll fix it up 😄